Answers.md

```diff
@@ -1,9 +1,43 @@
 1. If you had to describe semantic HTML to the next cohort of students, what would you say?
 
+### Semantic HTML
+#### It provides precise meaning to your content. In other words, we describe tags with meaning.
+#### For example:
+
+#### <header>
+#### <footer>
+#### <main>
+#### <section>
+
+##### Link: https://developer.mozilla.org/en-US/docs/Glossary/Semantics
+
+#### To better understand semantics, it makes your websites readable and easily interpreted by the screen-readers and search-engines. As a fellow developer, it helps the next developers to read and understand the code and easier to maintain as it is organized. 
+
+
 2. Describe some differences between ```display: block;``` and ```display: inline;```.
 
+### Differences between ```display: block;``` and ```display: inline;```
+
+### Inline-block
+#### It respects height and aligns next to each other. It allows you to cut out the page into multiple parts and have block level elements inside of them without taking up the entire page.
+
+#### Ex.: https://codepen.io/iteldomingo/pen/pqooMJ?editors=1100 
+
+### Block element
+#### Takes up the entire length of the page.
+#### Ex.: https://codepen.io/iteldomingo/pen/dwyPPm
+
+
 3. What are the 4 areas of the box model?
 
+#### Content, Margin, Border, Padding
+
+
 4. While using flexbox, what axis are you using when you use the property: ```align-items: center```?
 
-5. What is the git command to commit staged changes as well as write a message? 
\ No newline at end of file
+#### It controls alignment of all items on the cross axis.
+
+
+5. What is the git command to commit staged changes as well as write a message? 
+
```

Comment:
4/5 correct. The last question is asking what you type in the terminal to commit your changes and write a message?

Reply:
I've updated the answer. Thank you!